server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -743,13 +756,26 @@ private void stopBeingLeader()
     // CompactSegmentsDuty should be the last duty as it can take a long time to complete
     duties.addAll(makeCompactSegmentsDuty());
 
-    log.debug(
+    log.info(
```

Comment:
Is change intentional? If so, can you describe why it is better to be info? I think this was changed to debug to reduce the amount of coordinator logs.

Reply:
Changed back to debug.